### PR TITLE
doc / ongoing docs + release notes v1.0

### PR DESCRIPTION
**Pull request by JeremyKono**
1st commit:

Removed pnl, trades, open_orders and binance markets

Before:

image

After:

image

### CODE REVIEW

**Comment by JeremyKono** [comment]
2nd and 3rd commit:

Added release notes

image
image
image

File: docs/developers/connectors/index.md [hunk 1]
```diff
@@ -4,38 +4,24 @@ Exchange connectors are packages of code that link Hummingbot's internal trading
...
-
-- [Ndax](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/ndax)
-- [Crypto.com](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/crypto_com) 
-
-### Perpetual Connectors
-
-- [Bybit Perpetual](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/derivative/bybit_perpetual)
+We recommend referring to the [Binance Spot](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/binance)
+and the [Binance Spot](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/binance)
+connectors as the gold standard and most up-to-date implementations in the code base.
```
Comment (mifeng) [inline docs/developers/connectors/index.md:16]: - Add something similar telling devs building perp connectors to use Binance Perpetual

File: docs/release-notes/index.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ hide:
 
 We generally release a new version of Hummingbot every 4 weeks, along with periodic intermittent releases.
 
-The latest stable release is **[0.46.0](/release-notes/0.46.0)**, released on December 1, 2021.
\ No newline at end of file
+The latest stable release is **[1.0](/release-notes/1.0)**, released on January 24, 2021.
```
Comment (mifeng) [inline docs/release-notes/index.md:10]: For the release number, use `1.0.0` to follow prior convention

File: docs/release-notes/1.0.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Release Notes - Version 1.0
```
Comment (mifeng) [inline docs/release-notes/1.0.md:1]: Use `1.0.0` to follow past convention

**Review by mifeng — APPROVED** [review]
Looks good to me

**Comment by JeremyKono** [comment]
Changes made in the latest commits

**Comment by JeremyKono** [comment]
Added back the `Issues` column so that the users can better see the issues of a certain connector:

image

**Review by mifeng — APPROVED** [review]

**Pull request merged**